src/main/webapp/util/uccConfigurationValidators.js

```diff
@@ -64,6 +64,24 @@ export const parseFunctionRawStr = (rawStr) => {
     return { error, result };
 };
 
+export const parseFileValidator = (data, validFileTypes) => {
+    
+    const { fileName, fileSize, fileContent } = data;
+    const file_name = fileName.split(".");
+    const extension = "." + file_name[file_name.length - 1];
```

Comment:
I think we need to have just `json` instead of `.json`.

This code can be simplified to 
```
const givenFileExtension = fileName.split(".").pop();
```

So we don't need to have the manipulation with "." in the code here and we should ask user just to provide `json`, not `.json` in globalConfig file configuration.

Reply:
Agreed. I wanted to use the same convention same as the Splunk UI framework. because there are using something like this - `accept='.json'` in the File component.

But yes let's go with **json** instead of **.json** in the global config file.